client/components/data/query-stats-posts-views/README.md

```diff
@@ -0,0 +1,61 @@
+Query Posts Views
```

Comment:
I'm slightly worried this name implies the `stats/post-views` endpoint, rather than the new `stats/views/posts` endpoint we're using, but Query Views Posts just doesn't sound right 🤔

Reply:
Yeah, I switched back and forth about this. Honestly, it's most accurately "Query Recent Post Views" - given the num limitations on the endpoint. What do we think about changing it to that?

Reply:
Alternatively, similar to [your comment about the data structure](https://github.com/Automattic/wp-calypso/pull/27090#discussion_r217213476) below, we could refactor it to `QueryViews` and pass `posts` in a prop. It's more extendable for the future, but could just be over-optimization.